### PR TITLE
add bleJoystick as a button event

### PR DESCRIPTION
**Pull request by JyeSmith**
https://github.com/ExpressLRS/ExpressLRS/issues/2796

### CODE REVIEW

File: src/lib/BLE/devBLE.cpp [hunk 1]
```diff
@@ -71,6 +72,14 @@ void BluetoothJoystickBegin()
     bleGamepad->begin(gamepadConfig);
 }
 
+static void initialize()
+{
+  registerButtonFunction(ACTION_BLE_JOYSTICK, [](){
+    connectionState = bleJoystick;
+    devicesTriggerEvent();
```
Comment (CapnBry) [inline src/lib/BLE/devBLE.cpp:79]: This line is redundant, as devices.cpp will trigger the event automatically when connectionState changes. Triggering an event is pretty heavy so best to not double up.

Comment (JyeSmith) [reply src/lib/BLE/devBLE.cpp:79]: Ah nice.  I can also clean this up in the devwifi lib where I lifted it from.

**Review by CapnBry — APPROVED** [review]
Works great, tested with a Ranger Bling and it jumped right into Bluetoob mode. Could connect from my phone.

I approve but please do push the changes to remove the extra event generated.

**Review by pkendall64 — APPROVED** [review]

**Comment by VaSa4oK** [comment]
Congratulations. Is it possible to create a lua script to automatically turn on the ble joystick when selecting a model, or assign a specific toggle switch?

**Pull request merged**